tools/atlas/atlas.nim

```diff
@@ -812,6 +874,15 @@ proc main =
       echo toJson(extractRequiresInfo(args[0]))
     else:
       error "File does not exist: " & args[0]
+  of "tag":
+    projectCmd()
+    var field: SemVerField
+    if args.len == 0:
+      field = patch
+    else:
```

Comment:
Missing

``nim

elif '.' in args[0]:  
  field = args[0]

```

Except of course that it doesn't work because that is not an enum value...

Reply:
Oh for setting a custom tag? I can add that back, it's simple